### DISCUSSION TITLE
Syncing Firefox to Nextcloud Bookmarks: Why are there multiple folders with the same name?

### DISCUSSION THREAD

**Discussion by johanneskastl** [floccusaddon/floccus#1764] [Q&A] [answered]
Hi all,

I am trying to sync my Firefox Bookmarks to Nextcloud Bookmarks. I noticed that there are multiple folders/directories with the same name. I have "Bookmarks Bar" twice, "Bookmarks Menu" twice, etc.

Not sure what causes this, but I would have assumed that there can only be one folder of a certain name?

Kind Regards,
Johannes

**Comment by marcelklehr** [comment, marked as answer]
In browsers (and thus for compatibility reasons in Nextcloud Bookmarks), there can be multiple folders with the same name. That's just how it is.

**Reply by johanneskastl** [reply]
Hmm, I had assumed that Nextcloud Bookmarks (and thus Floccus) handle this in a more user-friendly way. But if that comes from Nextcloud Bookmarks I guess it is moot to complain... :-)

**Reply by marcelklehr** [reply]
It comes from browsers :) You are free to complain about Nextcloud Bookmarks, too, though, I'm also maintaining that one ;)

**Reply by johanneskastl** [reply]
Oops, sorry, it comes from the browsers. NC Bookmarks and Floccus are totally innocent. :-)

**Reply by marcelklehr** [reply]
The reason for the duplication might lie in floccus, the fact that they can coexist with the same name is a browser feature :D
